.travis.yml

```diff
@@ -177,16 +177,12 @@ script:
     else
         CFLAGS="--coverage" python setup.py build --build-lib build/lib/ --build-temp build/tmp/
         CFLAGS="--coverage" pip install -e . -v
-        pushd demo
-        pytest --pyargs pyedflib --cov=pyedflib --cov-config=../.coveragerc
-        cp .coverage ..
-        popd
+        python -m pytest --pyargs pyedflib --cov=pyedflib --cov-config=.coveragerc --cov-report=xml --verbose
     fi
 
 after_success:
   - |
     if [ "${MANYLINUX}" == "0" ]; then
-        codecov
```

Comment:
sure this is correct?

Reply:
yes, `codecov` only gathers the coverage report and submits it to codecov:

> Find coverage reports for all the languages below, gather them and submit them to Codecov.

source: https://github.com/codecov/codecov-python

The same is achieved by the small bash snippet a few lines below: 

> bash <(curl -s https://codecov.io/bash)

Last time I checked, that bash snippet was the recommended mode of uploading reports to codecov.

Removing the python `codecov` package here just prevents duplication.

Reply:
thanks!